File: RefugeUA.WebApp/refugeua.webapp.client/src/app/core/models/education-announcement.ts
import { TimeSpan } from "../../shared/util/time-span";
import { Announcement } from "./announcement";

export interface EducationAnnouncement extends Announcement {
    educationType: string,
    institutionName: string,
    targetGroups: string[],
    isFree: boolean,
    fee?: number,
    duration: number,
    language: string,
}

File: RefugeUA.Entities/EducationAnnouncement.cs
using RefugeUA.Entities.Abstracts;
using System;
using System.Collections.Generic;

namespace RefugeUA.Entities
{
    public class EducationAnnouncement : Announcement
    {
        public string EducationType { get; set; } = default!;

        public string InstitutionName { get; set; } = default!;

        public string TargetGroup { get; set; } = default!;

        public bool IsFree { get; set; }

        public decimal? Fee { get; set; }

        public int Duration { get; set; } = default!;

        public string Language { get; set; } = default!;
    }
}
